src/app/search/components/patient_search/patient_search.directive.spec.js

```diff
@@ -173,7 +159,13 @@
             vm.query.familyName =  'last';
             vm.searchForPatient();
             expect(commonService.searchForPatient).not.toHaveBeenCalled();
-            vm.query.dob = mock.dob;
+            vm.query.dob.year = mock.dob.year;
+            vm.searchForPatient();
+            expect(commonService.searchForPatient).not.toHaveBeenCalled();
+            vm.query.dob.month = mock.dob.month;
+            vm.searchForPatient();
+            expect(commonService.searchForPatient).not.toHaveBeenCalled();
+            vm.query.dob.day = mock.dob.day;
```

Comment:
I don't think there's a way for these tests to fail. As in, it'd be impossible for the searches to have been kicked off without the gender code below anyway, but this is probably not the place to fix that, unless you want to ;)